Gemfile

```diff
@@ -16,6 +16,7 @@ group :jekyll_plugins do
   gem 'jekyll-sitemap', '1.4.0'
   gem 'jekyll-time-to-read', '0.1.2'
   gem 'jekyll-commonmark', '1.3.1'
+  gem 'jekyll-toc'
```

Comment:
Please pin versions.

Reply:
Yep, I had this pinned in another branch, but I had to base off `current` and rebase, fixed now.